recipes/linalg-complex/24.11/gh200/environments.yaml

```
@@ -10,6 +10,7 @@ gcc-env:
   - arpack-ng
   - boost +atomic +chrono +container +context +coroutine +date_time +filesystem +iostreams +json +log +mpi +multithreaded +program_options +python +random +regex +serialization +shared +system +test +thread +timer +url
   - cmake
+  - dla-future +miniapps +mpi_gpu_aware +mpi_gpu_force_contiguous +scalapack
```

Add the fortran bindings:
```suggestion
  - dla-future +miniapps +mpi_gpu_aware +mpi_gpu_force_contiguous +scalapack
  - dla-future-fortran
```